docker-compose.yml

```diff
@@ -40,6 +40,26 @@ services:
       - "8086:8086"
     entrypoint: "gcloud beta emulators bigtable start"
 
+  learning_service:
+    build:
+      context: ./learning_service
+      dockerfile: Dockerfile
+    image: learning_service:0
+    container_name: learning_service
+    volumes:
+    - ./learning_service:/root
+    restart: unless-stopped
+
+  interface_service:
+    build:
+      context: ./interface_service
+      dockerfile: Dockerfile
+    image: interface_service:0
+    container_name: interface_service
+    ports:
+      - 8000:8000
+    restart: unless-stopped
```

Comment:
`unless-stopped` should be used carefully and definitely not for running the local mini-prod environment - imagine shutting down your computer, starting it later and wondering what is occupying that port 8000 even though the computer was only just started. Or what is hogging so much CPU all of the sudden.